src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -54,11 +54,11 @@
     <Authors>Microsoft</Authors>
     <Owners>microsoft,dotnetframework</Owners>
     <Description>TODO</Description>
-    <LicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</LicenseUrl>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>
```

Comment:
If you want to be careful you could just remove the default from the buildtools targets and force repos consuming buildtools to define it.  That way someone won't accidentally ship with the wrong license.

We're already doing that for the file itself, just not the URL apparently.
https://github.com/dotnet/corefx/blob/master/pkg/dotnet_library_license.txt 
https://github.com/dotnet/corefx/blob/master/Packaging.props#L5